tinder.js

```diff
@@ -339,6 +372,62 @@ function TinderClient() {
       makeTinderCallback(callback));
   };
 
+
+  /**
+   * Post new user picture
+   * @param {Buffer} buffer picture that you want to upload
+   * @param {Function} callback the callback to invoke when the request completes
+   */
+  this.uploadPicture = function(buffer, callback) {
+    tinderMultipartPost('image?client_photo_id=ProfilePhoto' + new Date().getTime() ,
+      {
+        userId : _this.userId,
+        file : buffer
+      },
+      makeTinderCallback(callback));
+  };
+
+  /**
+   * Post new user picture
+   * @param {String} id is the facebook id of the picture
+   * @param {Float} xdistance_percent is the zoom percentage in x 0 full Zoom 1 no Zoom
+   * @param {Float} ydistance_percent is the zoom percentage in x 0 full Zoom 1 no Zoom
+   * @param {Float} xoffset_percent is the offset from the left corner in percentage
+   * @param {Float} yoffset_percent is the offset from the top corner in percentage
+   * @param {Function} callback the callback to invoke when the request completes
+   */
+
+  this.uploadFBPicture = function(id, xdistance_percent, ydistance_percent, xoffset_percent, yoffset_percent, callback) {
+    tinderPost('media',
+      {
+        "transmit": "fb",
+        "assets": [{
+          "ydistance_percent": ydistance_percent,
+          "id": id,
+          "xoffset_percent": xoffset_percent,
+          "yoffset_percent": yoffset_percent,
+          "xdistance_percent": xdistance_percent
+        }]
+      },
+      makeTinderCallback(callback));
+  };
+
+  /**
+   * Delete picture from user profile
+   * @param {String || String Array} pictureIds the id(s) of the picture(s)
+   * @param {Function} callback the callback to invoke when the request completes
+   */
+  this.deletePictures = function(pictureIds, callback) {
+    if( typeof pictureIds === 'string' ) {
+      pictureIds = [ pictureIds ];
+    }
+    tinderDelete('media',
+      {
+        assets : pictureIds
+      },
+      makeTinderCallback(callback));
+  };
```

Comment:
Was this tested? Does the API actually delete multiple picture IDs when supplied in an array?

If so, we should change this to just accept an array, and leave it to the client to pass one in (even if there's just one record).